### DISCUSSION TITLE
Linux, Artisan, Huky Roaster and an old Phidget

### DISCUSSION THREAD

**Discussion by tomhusband** [artisan-roaster-scope/artisan#1177] [General]
I've been using a Win 10 laptop and Artisan for my Huky roaster for several years now but I want to move to Linux Mint (I'm slowly moving all my devices to Linux). I don't foresee any major issues with Artisan but the old 1048 Phidget that I've been using forever might be a different story. Would I be able to continue to use that Phidget? Are Linux drivers still available? I looked on the Phidget site but I'm not clear what's available.

Many thanks for any insight.

**Comment by tomhusband** [comment]
I'm going to pursue this with Phidget directly I think.

**Reply by MAKOMO** [reply]
Yes, this is better. Linux drivers are available. Not sure if those run on your specific variant (Linux Mint), but I guess.
